content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -1,89 +1,123 @@
+quality checks for that operation, rather than the manufacturing operation in general.
+
+For example, a |QCP| could be configured to create quality checks for the *Assembly* work order of
+the *Coffee Table* product. When a new |MO| is confirmed for a *Coffee Table*, the |QCP| creates a
+quality check for the *Assembly* operation specifically.
+
+The :guilabel:`Control Per` field is set to one of three options that determine *when* a new quality
+check is created:
+
+- :guilabel:`Operation`: one check is requested for the specified operation as a whole.
+- :guilabel:`Product`: one check is requested for each *unique* product included in the specified
+  operation. For example, a delivery operation for one table and four chairs would generate two
+  checks, since two *unique* products are included in the operation.
+- :guilabel:`Quantity`: a check is requested for a certain percentage of the items within the
+  specified operation. This percentage is set by enabling the :guilabel:`Partial Transfer Test`
+  checkbox, and then entering a numerical value in the :guilabel:`Percentage` field that appears
+  below. If the checkbox is not enabled, one quality check is created for the full quantity.
+
+The :guilabel:`Control Frequency` field is set to one of three options that determine *how often* a
+new quality check is created:
+
+- :guilabel:`All`: a quality check is requested every time the conditions of the |QCP| are met.
+- :guilabel:`Randomly`: a quality check is randomly requested for a certain percentage of
+  operations, which can be specified in the :guilabel:`Every #% of Transfers` field that appears
+  below.
+- :guilabel:`Periodically`: a quality check is requested once every set period of time, which can be
+  specified by entering a numerical value and choosing either :guilabel:`Days`, :guilabel:`Weeks`, or
+  :guilabel:`Months` in the two fields that appear below.
+
+In the :guilabel:`Type` field, specify the type of quality check that should be performed. The
+method for processing quality checks created by the |QCP| depends upon the type of quality check
+selected:
+
+- :guilabel:`Instructions` checks provide specific instructions for how to complete the quality
+  check.
+- :guilabel:`Take a Picture` checks require a picture of the product be uploaded for later review by
+  the assigned quality team.
+- :guilabel:`Register Production` checks prompt manufacturing employees to confirm the quantity of
+  the product that was produced during the manufacturing operation.
+- :guilabel:`Pass - Fail` checks specify a criterion that products must meet for the check to pass.
+- :guilabel:`Measure` checks prompt employees to record a measurement of the product that must be
+  within a tolerance of a norm value for the check to pass.
+- :guilabel:`Worksheet` checks provide an interactive worksheet that must be filled out by the
+  employee processing the check.
```

Comment:
Totally optional, but I'd personally also add links to each of the quality check type docs that you wrote, so people can read more about them on another page (: